I found a bug using my cypher fuzzer.

When running the following query against an empty database:
```cypher
CREATE (:p9VhzSI{rp8di:(false)}), (x:p9VhzSI{rp8di:(true)})-[maH3gC:Ch2pkxty1{iMF:null}]->(x) RETURN maH3gC AS vINcVm3 UNION WITH  (null) AS DK, toBoolean(("J{_")) AS xIp   LIMIT (4934590114617846107) OPTIONAL MATCH (WKFKfF17:p9VhzSI{kG0wnyNFO:(9223372036854775807)})-[LbR:Ch2pkxty1*..0{iMF:(379401289610449505), kG0wnyNFO:(false), kG0wnyNFO:WKFKfF17.kG0wnyNFO}]->() WHERE (false) WITH  ([]+null) AS zBWJE9fk_E, [("")] AS vmk   LIMIT (8381277545231420015)  CREATE wTDASSTxD = (:p9VhzSI)-[UEIBX0P:Ch2pkxty1]->(:p9VhzSI{b_TVcJ:(1206446281497874016), C8FMT:(-1555143608682847049)})-[:Ch2pkxty1{VWjX:(true)}]->(:p9VhzSI{y1MWrcJ:("~kJ,"), kG0wnyNFO:(null)})-[eTiiv9:Ch2pkxty1{kG0wnyNFO:sqrt((-1.0)), oUVPTKk5:(-6.455914787462944e-188), kG0wnyNFO:(1142040223725829132)}]->(pfPToaxT:p9VhzSI)<-[v:Ch2pkxty1{rp8di:(4745767364729720215)}]-(rThQ8:p9VhzSI{yARIVeE:null, mQheJeUI:(false), kG0wnyNFO:(null)}) RETURN pfPToaxT AS vINcVm3
```

I encountered the following error:
```
pq: invalid range
```

I believe the query mentioned above is semantically and syntactically correct and thus no error should be thrown here.

Additionally, the query runs successfully in neo4j.

I encountered this issue when testing queries against the **apache/age:PG13_latest** docker image.

### Steps to reproduce

Spin up a local instance of **apache/age:PG13_latest**: `docker run -e POSTGRES_PASSWORD=123 --rm --name age apache/age:PG13_latest`

Get a shell in the docker container: `docker exec -it age /bin/bash`

Connect to postgres: `su postgres -c psql`

Run the following queries:
```psql
LOAD 'age';
---
SET search_path = ag_catalog, "$user", public;
---
SELECT create_graph('graph');
---
SELECT * FROM cypher('graph',$$
	CREATE (:p9VhzSI{rp8di:(false)}), (x:p9VhzSI{rp8di:(true)})-[maH3gC:Ch2pkxty1{iMF:null}]->(x) RETURN maH3gC AS vINcVm3 UNION WITH  (null) AS DK, toBoolean(("J{_")) AS xIp   LIMIT (4934590114617846107) OPTIONAL MATCH (WKFKfF17:p9VhzSI{kG0wnyNFO:(9223372036854775807)})-[LbR:Ch2pkxty1*..0{iMF:(379401289610449505), kG0wnyNFO:(false), kG0wnyNFO:WKFKfF17.kG0wnyNFO}]->() WHERE (false) WITH  ([]+null) AS zBWJE9fk_E, [("")] AS vmk   LIMIT (8381277545231420015)  CREATE wTDASSTxD = (:p9VhzSI)-[UEIBX0P:Ch2pkxty1]->(:p9VhzSI{b_TVcJ:(1206446281497874016), C8FMT:(-1555143608682847049)})-[:Ch2pkxty1{VWjX:(true)}]->(:p9VhzSI{y1MWrcJ:("~kJ,"), kG0wnyNFO:(null)})-[eTiiv9:Ch2pkxty1{kG0wnyNFO:sqrt((-1.0)), oUVPTKk5:(-6.455914787462944e-188), kG0wnyNFO:(1142040223725829132)}]->(pfPToaxT:p9VhzSI)<-[v:Ch2pkxty1{rp8di:(4745767364729720215)}]-(rThQ8:p9VhzSI{yARIVeE:null, mQheJeUI:(false), kG0wnyNFO:(null)}) RETURN pfPToaxT AS vINcVm3
$$) as (v agtype);
---
CREATE (:p9VhzSI{rp8di:(false)}), (x:p9VhzSI{rp8di:(true)})-[maH3gC:Ch2pkxty1{iMF:null}]->(x) RETURN maH3gC AS vINcVm3 UNION WITH  (null) AS DK, toBoolean(("J{_")) AS xIp   LIMIT (4934590114617846107) OPTIONAL MATCH (WKFKfF17:p9VhzSI{kG0wnyNFO:(9223372036854775807)})-[LbR:Ch2pkxty1*..0{iMF:(379401289610449505), kG0wnyNFO:(false), kG0wnyNFO:WKFKfF17.kG0wnyNFO}]->() WHERE (false) WITH  ([]+null) AS zBWJE9fk_E, [("")] AS vmk   LIMIT (8381277545231420015)  CREATE wTDASSTxD = (:p9VhzSI)-[UEIBX0P:Ch2pkxty1]->(:p9VhzSI{b_TVcJ:(1206446281497874016), C8FMT:(-1555143608682847049)})-[:Ch2pkxty1{VWjX:(true)}]->(:p9VhzSI{y1MWrcJ:("~kJ,"), kG0wnyNFO:(null)})-[eTiiv9:Ch2pkxty1{kG0wnyNFO:sqrt((-1.0)), oUVPTKk5:(-6.455914787462944e-188), kG0wnyNFO:(1142040223725829132)}]->(pfPToaxT:p9VhzSI)<-[v:Ch2pkxty1{rp8di:(4745767364729720215)}]-(rThQ8:p9VhzSI{yARIVeE:null, mQheJeUI:(false), kG0wnyNFO:(null)}) RETURN pfPToaxT AS vINcVm3
```

### Expected behavior
The query should run successfully

### Actual behavior
The query fails with the error message `pq: invalid range`.
